### PR TITLE
Add prettier and eslint configs

### PR DESCRIPTION
**Pull request by sndrs**
creates a lerna monorepo with 2 packages:

- prettier config (`@guardian/prettier`)
- eslint config (`@guardian/eslint-config`)

and uses them in the repo itself

### CODE REVIEW

**Comment by JamieB-gu** [comment]
> `.yarn/releases/yarn-1.22.10.js`

Is this the whole of Yarn...?

**Comment by sndrs** [comment]
> Is this the whole of Yarn...?

yeah, pins the version of yarn you're using. any local yarn>v1 will use it

File: packages/prettier/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+module.exports = {
+	arrowParens: 'always',
+	bracketSpacing: true,
+	jsxBracketSameLine: false,
+	jsxSingleQuote: false,
+	printWidth: 80,
+	quoteProps: 'as-needed',
+	semi: false,
```
Comment (gtrufitt) [inline packages/prettier/index.js]: 👀 ;

**Review by gtrufitt — APPROVED** [review]
This is really cool +1

**Review by SiAdcock — APPROVED** [review]
There's so much I have learned from this one PR! 🌟 

And I don't feel the need to disagree with any of it! 🤗 

Amazing stuff @sndrs 💎 👍

File: packages/prettier/index.js (hunk 1 above)
Comment (sndrs) [reply packages/prettier/index.js]: yeah i think is not a good idea - will mean loads of codebases will need to reformat and it doesn't fix anything. will change it

Comment (gtrufitt) [reply packages/prettier/index.js]: Aces 👍 I think everything else is a fair-enough non-default!

**Pull request merged**